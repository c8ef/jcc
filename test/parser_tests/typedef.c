int main() {
  typedef int A, B;
  A a = 42;
  B b = 24;
}
