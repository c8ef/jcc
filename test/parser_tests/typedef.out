FunctionDecl: main
  Args[0]
  CompoundStatement
    DeclStatement
      VarDecl: a
        IntergerLiteral: 42
    DeclStatement
      VarDecl: b
        IntergerLiteral: 24
